src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -255,6 +255,13 @@ enum class ConfigType : uint16_t {
    */
   USER_DEVICE_ID = 301,
 
+  /**
+   * A bitmask indicating which profiling features are enabled.
+   *
+   * Payload format: `uint8_t` (see @ref sat_type_masks)
```

Comment:
Should this be a uint32 for expandability?

Reply:
I considered this, but:
1. The device side implementation is much simpler when there's a 1-1 mapping
2. I don't want the actual profiling details to be part of the public standard which also is simpler if its a 1-1 mapping
3. It's trivial to add a new type if we want to have a more transparent setting